conan/tools/qbs/qbs.py

```diff
@@ -20,8 +20,7 @@ def _configuration_dict_to_commandlist(name, config_dict):
 
 class Qbs(object):
     def __init__(self, conanfile, project_file=None):
-        # hardcoded name, see qbs toolchain
-        self.profile = 'conan_toolchain_profile'
+        self.profile = None
```

Comment:
Ok, if I understood it correctly, this is a provision for when the ``QbsToolchain`` generates this file and in that case it will be automatically used, but that logic will be done together with ``QbsToolchain``